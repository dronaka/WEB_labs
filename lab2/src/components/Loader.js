import React from "react";
import "./styles/Loader.css";


export default function Loader() {
  return (
    <div className="loader">
      <p>Forecast is loading...</p>
    </div>
  );
}