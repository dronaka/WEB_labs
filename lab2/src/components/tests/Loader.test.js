import React from "react";
import renderer from "react-test-renderer";

import Loader from "../Loader";

describe("<Loader /> render", () => {

  test("should has text", () => {
    const tree = renderer
      .create(<Loader />)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
