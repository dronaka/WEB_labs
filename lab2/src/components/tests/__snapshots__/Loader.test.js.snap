// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loader /> render should has text 1`] = `
<div
  className="loader"
>
  <p>
    Forecast is loading...
  </p>
</div>
`;
